rubocop-rspec_rails.gemspec

```diff
@@ -32,12 +29,11 @@ Gem::Specification.new do |spec|
   spec.extra_rdoc_files = ['MIT-LICENSE.md', 'README.md']
 
   spec.metadata = {
-    'changelog_uri' => 'https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md',
-    'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec/',
+    'changelog_uri' => 'https://github.com/rubocop/rubocop-rspec_rails/blob/master/CHANGELOG.md',
+    'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec_rails/',
     'rubygems_mfa_required' => 'true'
   }
 
   spec.add_runtime_dependency 'rubocop', '~> 1.40'
-  spec.add_runtime_dependency 'rubocop-capybara', '~> 2.17'
-  spec.add_runtime_dependency 'rubocop-factory_bot', '~> 2.22'
+  spec.add_runtime_dependency 'rubocop-rspec', '~> 2.27.1'
```

Comment:
Should we make it depend on 2.28.0 where all cops are extracted to prevent duplicates? I can’t recall how we did this in the past.
Point to github from Gemfile to ensure edge, release this gem first, then release rubocop-rspec 2.28.0, then change this gemfile to remove github source and point to rubygems.
I haven’t had my morning coffee yet, so please take this idea with a grain of salt.

Reply:
In the past we didn't need to worry about it, since rubocop-capybara and rubocop-factory_bot didn't rely on rubocop-rspec. Sorry. I want you to tell me, what kind of problem do I have if I don't depend on it after it's extracted?

Reply:
In theory, it would be possible to install rubocop-rspec_rails 2.28.0 with rubocop-rapec 2.27.1, where the extraction didn’t happen yet.
It could happen, but seems unrealistic.
Other cases would be like the one we’ve had when we wanted to yank a certain version? In some circumstances, bundler can prefer to downgrade rubocop-rspec to this combination of versions.
https://github.com/rubocop/rubocop-rspec/pull/1567

And, we should lick this to <3.0 here

Reply:
Thank you. Indeed it is. I update this PR https://github.com/ydah/rubocop-rspec_rails/pull/1/commits/1f412fd0016ac1d51d002db33f4fe8965b30fac9